project_members.go

```diff
@@ -117,6 +117,31 @@ func (s *ProjectMembersService) GetProjectMember(pid interface{}, user int, opti
 	return pm, resp, err
 }
 
+// GetAllProjectMember gets a project team member, including inherited
```

Comment:
This needs to change as well now:
```suggestion
// GetInheritedProjectMember gets a project team member, including inherited.
```

Reply:
Yup, missed it, thanks.